scripts/update-dockerhub-description.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+image_name="all-in-one"
+readme_path="./README.md"
+token="$DOCKERHUB_TOKEN"
+repository="jaegertracing/$image_name"
+url="https://hub.docker.com/v2/repositories/$repository/"
+readme_content=$(cat $readme_path)
+payload="{\"full_description\":\"$readme_content\"}"
+
+response=$(curl -s -X PATCH \
+  -H "Authorization: JWT $token" \
```

Comment:
with `set -x` this will cause the token to be printed to the logs, we cannot allow that.